pybossa/view/projects.py

```diff
@@ -3142,6 +3142,9 @@ def coowners(short_name):
     for owner, p_owner in zip(owners, pub_owners):
         if owner.id == project.owner_id:
             p_owner['is_creator'] = True
+    contact_owners = [user for user in owners if user.enabled and (user.id == project.owner_id or user.admin or user.subadmin)]
```

Comment:
How about creating an inner function to handle ```user in owners if user.enabled and (user.id == project.owner_id or user.admin or user.subadmin``` so that it can be reused for line 3985?

Reply:
Fixed via https://github.com/bloomberg/pybossa/pull/761/commits/b3dbc4b7f57b8527a01aff4edf3830fcc9877f2b